src/demo-app/a11y/radio/radio-a11y.html

```diff
@@ -0,0 +1,32 @@
+<section>
+  <h2>Radio buttons in group</h2>
+  <md-radio-group name="seasons" [(ngModel)]="favoriteSeason">
+    <md-radio-button *ngFor="let season of seasonOptions" [value]="season">
+      {{season}}
+    </md-radio-button>
+  </md-radio-group>
+</section>
+
+<section>
+  <h2>Radio buttons with align-end label position</h2>
+  <md-radio-group name="my_options" align="end">
+    <md-radio-button value="option_1">Option 1</md-radio-button>
+    <md-radio-button value="option_2">Option 2</md-radio-button>
+    <md-radio-button value="option_3">Option 3</md-radio-button>
```

Comment:
It would be good to use more meaningful options (like the seasons example does). Good examples could be pizza toppings, ice cream flavors, varieties of tea, types of bread, etc.